assets/js/app.js

```diff
@@ -24,7 +24,41 @@ let header = $(`
 </div>
 </nav>`);
 
-let footer = $(`
```

Comment:
Please do not change the files not related to the task your assigned to complete